src/oceans/i18n.js

```diff
@@ -0,0 +1,12 @@
+import data from '../../i18n/oceans.json';
+import MessageFormat from 'messageformat'
+
+let messages;
+export const init = (i18n = {}) => {
+  const mf = new MessageFormat('en');
```

Comment:
I don't love that we're hardcoding 'en' here even when we're actually rendering in another language; but I also don't see that this matters enough for us to go through all the work of passing locale in from code-dot-org.

Could you maybe just add a comment here explaining why we're doing what we're doing?